### ISSUE TITLE
a "for dummies" sort of version for people who aren't so inclined toward learning by reading

### ISSUE THREAD

**Issue by codejack42** [flattenthecurve/guide#72]
I've shared the "Update and Thorough Guidance" doc with a bunch of people. Among those who are natural readers, it gets high marks uniformly. But those who are less inclined to read tend to not read it, or to only get through the first section or two, or to just skim and forget. I asked a number of people for feedback and suggestions, and a few of them said it was "too dense", "too wordy", "not visual enough", "written at a high enough level so that a lot of people will find it off-putting". I received a couple suggestions for something that is more visual and when I said "You mean something like a 'for dummies' version?" the answer came back a resounding yes.

**Comment by nditada** [comment]
I like this, let's build a "Summary" section with the "too long;didn't read". We would need to craft that in english so that all translators can then take it from there.

**Comment by drlbln** [comment]
As I said in the translation-ops channel on slack: I strongly support this. This really is urgent.
While here in Europe (partial) shutdowns and lockdowns are reality already, still a lot of people don't understand simple basics. And particularly those with little or no English skills are the most affected.  So, please let us bring up a short maybe boild down version soon/now.  And though I know it's good scientific practice to have independet review by domain experts we have to find a way to make contribution easier/ more dynamic. Speed is key. And a short and  crisp version is  kind of a translation and should could be handled as one.
